server/handler/base.go

```diff
@@ -122,7 +125,7 @@ func (b *Base) PreparePRContext(ctx context.Context, installationID int64, pr *g
 	return ctx, logger
 }
 
-func (b *Base) Evaluate(ctx context.Context, installationID int64, loc pull.Locator) error {
+func (b *Base) Evaluate(ctx context.Context, installationID int64, performActions bool, loc pull.Locator) error {
```

Comment:
I don't feel great about this `performActions` parameter, but I'm also not sure how to make it better without duplicating a lot of code or doing a big refactor.

Reply:
:/ this is how I felt as well.